[2025-05-23T11:41:28.911-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-23T11:41:28.949-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: libertyPr_health_score.send_to_elasticsearch scheduled__2025-05-22T00:00:00+00:00 [queued]>
[2025-05-23T11:41:28.966-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: libertyPr_health_score.send_to_elasticsearch scheduled__2025-05-22T00:00:00+00:00 [queued]>
[2025-05-23T11:41:28.966-0600] {taskinstance.py:2867} INFO - Starting attempt 2 of 3
[2025-05-23T11:41:28.984-0600] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): send_to_elasticsearch> on 2025-05-22 00:00:00+00:00
[2025-05-23T11:41:28.995-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=930) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-23T11:41:28.998-0600] {standard_task_runner.py:72} INFO - Started process 932 to run task
[2025-05-23T11:41:28.999-0600] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'libertyPr_health_score', 'send_to_elasticsearch', 'scheduled__2025-05-22T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/health_score_lpr.py', '--cfg-path', '/tmp/tmpsfockhad']
[2025-05-23T11:41:29.004-0600] {standard_task_runner.py:105} INFO - Job 25: Subtask send_to_elasticsearch
[2025-05-23T11:41:29.093-0600] {task_command.py:467} INFO - Running <TaskInstance: libertyPr_health_score.send_to_elasticsearch scheduled__2025-05-22T00:00:00+00:00 [running]> on host 439130e6349d
[2025-05-23T11:41:29.225-0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='libertyPr_health_score' AIRFLOW_CTX_TASK_ID='send_to_elasticsearch' AIRFLOW_CTX_EXECUTION_DATE='2025-05-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-22T00:00:00+00:00'
[2025-05-23T11:41:29.228-0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-23T11:41:31.303-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/elasticsearch/_sync/client/__init__.py:403: SecurityWarning: Connecting to 'https://10.212.56.57:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2025-05-23T11:41:31.736-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T11:41:32.105-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.377s]
[2025-05-23T11:41:32.122-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T11:41:32.233-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.111s]
[2025-05-23T11:41:32.250-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T11:41:32.327-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.078s]
[2025-05-23T11:41:32.342-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T11:41:32.401-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.059s]
[2025-05-23T11:41:32.418-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T11:41:32.479-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.062s]
[2025-05-23T11:41:32.494-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T11:41:32.547-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.053s]
[2025-05-23T11:41:32.566-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T11:41:32.627-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.061s]
[2025-05-23T11:41:32.636-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T11:41:32.658-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.023s]
[2025-05-23T11:41:32.661-0600] {logging_mixin.py:190} INFO - âœ… Sent 3603 documents to Elasticsearch index 'libertypr_health_score'
[2025-05-23T11:41:32.663-0600] {python.py:240} INFO - Done. Returned value was: None
[2025-05-23T11:41:32.689-0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-23T11:41:32.690-0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=libertyPr_health_score, task_id=send_to_elasticsearch, run_id=scheduled__2025-05-22T00:00:00+00:00, execution_date=20250522T000000, start_date=20250523T174128, end_date=20250523T174132
[2025-05-23T11:41:32.789-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-23T11:41:32.831-0600] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-23T11:41:32.834-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
